pagopa/io-app
# [SIW-675] Add QR Code screen during proximity flow

## Short description
This PR adds the QRCode screen during proximity flow (for testing purposes the qr-code content is mocked with a fake mDoc payload)

## List of changes proposed in this pull request
- Add ItwPresentationQrCodeScreen.tsx

## How to test
Open the credential details and tap "Show QR Code". The new screen should appear. For testing purposes it is possible to perform a proximity flow with a tap on the loading message.
